.gitignore

```diff
@@ -24,3 +24,7 @@ gon.json
 
 # We don't want to commit env variables
 *.env
+
+# IntelliJ
```

Comment:
[not-blocking] I personally prefer to set things like personal dev environments in a global gitignore but I think this is the second time someone tries to commit this so I'm open to make it official